### PR TITLE
Cilogon custom profile

### PR DESCRIPTION
**Pull request by ejdysinger**
verbiage added; I also changed how the IdP was evaluated so we could pass a parameter to the forbidden page

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -107,11 +107,17 @@ export default defineNuxtConfig({
             : "https://fairhub.io",
       ENTRA_CONFIG: {
         authority:
-          "https://aireadi.b2clogin.com/aireadi.onmicrosoft.com/B2C_1_ARVO_SignUpSignIn_Workflow",
+          "https://aireadi.b2clogin.com/aireadi.onmicrosoft.com/B2C_1A_TRUSTFRAMEWORKBASE_CILOGON_IDP_RESTRICTED", // https://aireadi.b2clogin.com/aireadi.onmicrosoft.com/B2C_1_ARVO_SignUpSignIn_Workflow",
```
Comment (heathharrelson) [inline nuxt.config.ts:110]: Did you intend to leave the commented code here?

File: server/routes/login.get.ts [hunk 2]
```diff
@@ -59,30 +64,38 @@ function getEmail(tokenResponse: AuthenticationResult): string {
...
 
-function convertTokenResponse(
-  tokenResponse: AuthenticationResult,
-): SessionUserDetails {
+async function convertTokenResponse(tokenResponse: AuthenticationResult) {
   const indexableClaims = { ...tokenResponse.idTokenClaims };
-  const id = getStringTokenClaim(indexableClaims, "oid");
+  const issuer = getStringTokenClaim(indexableClaims, "iss");
+  const subject = getStringTokenClaim(indexableClaims, "sub");
+  // const id = getStringTokenClaim(indexableClaims, "oid");
```
Comment (heathharrelson) [inline server/routes/login.get.ts:71]: This could be removed.

Comment (ejdysinger) [reply server/routes/login.get.ts:71]: you're right let me remove

File: nuxt.config.ts (hunk 1 above)
Comment (ejdysinger) [reply nuxt.config.ts:110]: negative, I will remove that

**Review by heathharrelson — APPROVED** [review]
Looks good to me, and the latest code is working locally. Thanks for all the work on this.

File: prisma/schema.prisma
Comment (kirenotneb) [inline prisma/schema.prisma:1]: @heathharrelson any concerns about the changes to the schema here?

**Review by kirenotneb — COMMENTED** [review]
I want to test logout and a couple things locally although I know heath has already done a pass at it.

File: prisma/schema.prisma
Comment (heathharrelson) [reply prisma/schema.prisma:1]: No, this looks how I expect.

**Review by kirenotneb — APPROVED** [review]
I successfully tested the full flow including accessing the forbidden login page through its various branches. It's working as expected. Thanks! Good work @ejdysinger . We should be able to merge and deploy to stage at this time.

**Pull request merged**